src/sentry/models/release.py

```diff
@@ -67,6 +69,75 @@ class ReleaseCommitError(Exception):
     pass
 
 
+def dynamic_sampling_has_latest_release(rules):
```

Comment:
Nit pick (feel free to ignore)
would rename to `ds_rules_contain_latest_release_rule`

Reply:
yeah will do.